src/Microsoft.HttpRepl.IntegrationTests/Mocks/ConsoleManagerStub.cs

```diff
@@ -0,0 +1,45 @@
+using System;
```

Comment:
nit: Missing copyrignt comment

Reply:
Fixed in https://github.com/aspnet/HttpRepl/pull/74/commits/af55ecc8092914f16ced3d5fd83db4b96b4de1de